curriculum/challenges/english/03-front-end-libraries/sass/extend-one-set-of-css-styles-to-another-element.english.md

```diff
@@ -76,8 +76,31 @@ tests:
 
 ## Solution
 <section id='solution'>
+The new class simples uses the ```@extend .info;``` class above and adds the ```background-color: magenta;```.
```

Comment:
Hi @paulywill

I think these comments will not be rendered in the UI. The solution is there to test against the tests in our CI. Meaning these are required for us to know that future edits to the challenge satisfy the solution.

You should remove them.